lib/streamlit/elements/arrow.py

```diff
@@ -30,14 +30,23 @@
 from pandas import DataFrame
 from pandas.io.formats.style import Styler
 
+import streamlit as st
 from streamlit import type_util
 from streamlit.proto.Arrow_pb2 import Arrow as ArrowProto
 from streamlit.runtime.metrics_util import gather_metrics
 
 if TYPE_CHECKING:
     from streamlit.delta_generator import DeltaGenerator
 
-Data = Union[DataFrame, Styler, pa.Table, ndarray, Iterable, Dict[str, List[Any]], None]
+Data = Union[
```

Comment:
nit: I think this could be formatted back to the existing formatting since nothing was actually changed here